### ISSUE TITLE
populateAccessToken tries to access non-existent property

### ISSUE THREAD

**Issue by timelesshaze** [feathersjs/feathers-authentication-client#11]
### Steps to reproduce

The function populateAccessToken fails to get the accessToken defined on the feathers application as hook.app is not the actual feathers app.

https://github.com/feathersjs/feathers-authentication-client/blob/ffefbc91512ef442dca2d18f0957735a2bf02898/src/hooks/populate-access-token.js#L8

hook.app is an object with the following keys:
* "app": the feathers application
* "service": the currently called service
* "path": a getter for the current service path

Test case (mixed import and require):
```javascript
import * as io from 'socket.io-client';
import * as RxJs from 'rxjs';

const feathers = require('feathers/client');
const feathersSocketio = require('feathers-socketio/client');
const feathersreactive = require('feathers-reactive');

const hooks = require('feathers-hooks');
const localstorage = require('feathers-localstorage');
const authentication = require('feathers-authentication-client');

const url = '...';
const socket = io(url);

const app = feathers()
  .configure(feathersSocketio(socket))
  .configure(hooks())
  .configure(authentication({ storage: window.localStorage }));

app.service('customers').find()
  .then(() => {})
  .catch((ex) => {
    console.error(ex); // TypeError: app.get is not a function(…)
   })
```

### Expected behavior
hook.app should be the feathers application or the method should use hook.app.app

### Actual behavior
The method dies while trying to call the non-existent get method

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
feathers@2.0.2
feathers-authentication-client@0.1.3
feathers-hooks@1.7.0
feathers-localstorage@1.0.0
feathers-reactive@0.4.1
feathers-socketio@1.4.2

**NodeJS version**:
v6.9.1

**Operating System**:
Ubuntu 16.04.1

**Browser Version**:
Google Chrome 54.0
Google Chrome 55.0
Firefox 50

**React Native Version**:
N/A

**Module Loader**:
What Angular2 2.2.4 ships with

**Comment by wgrafael** [comment]
Same problem and I use Angular2 too.

**Comment by wgrafael** [comment]
See this: https://github.com/feathersjs/feathers-commons/commit/4e104c1e947c0555b9b223ce855d2a8ab54de71e

I had same problem, so i updated the **feathers-commons** the problem was solved.

**Comment by daffl** [comment]
That's odd since that dependency should be installed automatically with `feathers-hooks` 1.7.0.

**Comment by wgrafael** [comment]
In feathers package, the feathers-commons dependencies is 0.8.0 (> = 0.80 <0.9).
Https://github.com/feathersjs/feathers/blob/master/package.json#L56

And if I install feathers package first, when install feathers-hooks, the common does not update. I do not know if npm should update the dependency or warning about this difference, but does nothing.

(I did a test: I created new folder and installed the feathers before feathers-hooks, and the feathers-commom does not has updated. After, I did it the other way, first  i installed feathers-hooks and after feathers and version was corrected.)
